workshops/templates/workshops/admin_dashboard.html

```diff
@@ -39,6 +40,8 @@
     <table class="table table-striped">
     {% for event in current_events %}
     <tr>
+      <td class="text-center" width="25px">{% if event.is_swc %}{% bootstrap_tag "SWC" %}{% endif %}</td>
+      <td class="text-center" width="25px">{% if event.is_dc %}{% bootstrap_tag "DC" %}{% endif %}</td>
```

Comment:
@chrismedrela if they need to be aligned, then please make more space for them on the dashboard (as @maneesha suggested in the original post).

As for the looping, it looks like it will yield a lot of queries, but let me check first. Also, currently you add code that's not being used in this PR.